tests/functional/disaster-recovery/regional-dr/test_dryrun_failover.py

```
@@ -0,0 +1,1034 @@
+                f"[{info['drpc_resource_name']}] Waiting for "
+                f"'{constants.STATUS_TESTING_FAILOVER}'"
+            )
+            info["drpc_obj"].wait_for_progression_status(
+                constants.STATUS_TESTING_FAILOVER
+            )
+            logger.info(
+                f"[{info['drpc_resource_name']}] Reached "
+                f"'{constants.STATUS_TESTING_FAILOVER}' ✓"
+            )
+
+    def _verify_dryrun_active(
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_verify_dryrun_active`

Add return type annotation: `None`

(ANN202)